submission/submission-webapp/src/test/kotlin/ac/uk/ebi/biostd/data/service/SubmissionDraftServiceTest.kt

```diff
@@ -0,0 +1,98 @@
+package ac.uk.ebi.biostd.data.service
+
+import ac.uk.ebi.biostd.persistence.common.request.PaginationFilter
+import ac.uk.ebi.biostd.persistence.model.DbUserData
+import ac.uk.ebi.biostd.submission.domain.service.SubmissionService
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import io.mockk.mockkStatic
+import io.mockk.unmockkStatic
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.extension.ExtendWith
+import java.time.Instant
+
+private const val USER_ID = 1234L
+private const val DRAFT_KEY = "key"
+private const val DRAFT_CONTENT = "data"
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@ExtendWith(MockKExtension::class)
+class SubmissionDraftServiceTest(
+    @MockK private val userDataService: UserDataService,
+    @MockK private val submissionService: SubmissionService
+) {
+    val testInstance = SubmissionSqlDraftService(userDataService, submissionService)
+
+    private val dbUserData = DbUserData(USER_ID, DRAFT_KEY, DRAFT_CONTENT)
+    private val someFilter: PaginationFilter = PaginationFilter()
+
+    @Test
+    fun `get submission when exists`() {
```

Comment:
Is this to get the draft when the submission exists? I think we should be as specific as possible with the name scenario since submission and draft are two different things